docs/addons/unleash.md

```diff
@@ -3,13 +3,29 @@
 > Unleash is a feature toggle system, that gives you a great overview of all feature toggles across all your applications and services. It comes with official client implementations for Java, Node.js, Go, Ruby, Python and .Net.
 
 We use [Unleash](https://docs.getunleash.io/) for feature toggling in NAIS and NAV.
-Get started at (unleash.nais.io)[https://unleash.nais.io/].
+Get started at [unleash.nais.io](https://unleash.nais.io/), and you can look at tips for your [feature toggles available here](https://www.unleash-hosted.com/articles/what-are-the-feature-toggle-best-practices).
 
-We only have one supported environment, but if you want to test it out you can use [unleash.dev.intern.nav.no](https://unleash.dev.intern.nav.no/).
+We've only got _one_  supported environment - but if you want to test it out you can use [(https://)unleash.dev.intern.nav.no](https://unleash.dev.intern.nav.no/).
 
-## Support
-Ask question in [#unleash](https://nav-it.slack.com/archives/C9BPTSULS).
+## NAV Recommended best practices
+
+!!! notice
+    The below tips are just a simplified digest [of tips listed on their official page](https://www.unleash-hosted.com/articles/what-are-the-feature-toggle-best-practices).
```

Comment:
Usikker på hvor _viktig_ det er at folk får med seg at listen nedenfor bare en simplifisert versjon av den offisielle dokumentasjonen. Er det ikke bedre at fokuset er på listen, også kan vi heller nevne det i en bisetning?

Reply:
For all del. Har ingen sterke meniger på dette.

Det står som det gjør fordi jeg tolker våre utviklere til å ha lyst på en liste skrevet av oss, ikke Unleash. 
Derav lagt på notice/caveat for de som gidder å gjøre mer enn absolutt minimum (som er de punktene jeg skrev her).

Har ingenting imot å fjerne listen - men synes ikke listen til Unleash er så lett å fordøye på 20 sekunder tbh.
Hva tenker du er best her?

Reply:
Var ikke listen jeg tenkte på, den syns jeg er fin. Var _notice_-panelet jeg tenkte på.

Reply:
Gjør noe med det om du ønsker =)

Har du ikke gjort noe (ingen sterke meninger her, sålenge vi gjør det lett for folk å komme seg dit på noe vis om de vil) innen jeg er tilbake fra ferie, merger jeg det inn sånn som det står^^

Reply:
Nah, gidder ikke :p Bare merger nå jeg, så får vi det ut :)